.Gruntfile.babel.js

```diff
@@ -179,7 +179,7 @@ module.exports = (grunt) => {
   grunt.registerTask('backend', ['backend:relaunch', 'watch'])
   grunt.registerTask('dev', ['checkDependencies', 'build:watch', 'connect', 'backend'])
   grunt.registerTask('dist', ['build'])
-  grunt.registerTask('test', ['dist', 'connect', 'exec:test'])
+  grunt.registerTask('test', ['dist', 'exec:test'])
```

Comment:
Note: Trying to make `grunt test` to run only backend tests without starting the server, but couldn't do it...

Reply:
Let's preserve the behavior of `grunt test` — it should run both backend and frontend tests.